urllib3/response.py

```diff
@@ -117,8 +122,17 @@ def __init__(self, body='', headers=None, status=0, version=0, reason=None,
         if hasattr(body, 'read'):
             self._fp = body
 
-        if preload_content and not self._body:
-            self._body = self.read(decode_content=decode_content)
+        # Are we using the chunked-style of transfer encoding?
+        self.chunked = False
+        self.chunk_left = None
+        tr_enc = self.headers.get('transfer-encoding')
```

Comment:
You can save yourself a conditional check on the next line by returning the empty string in the failure case:

```python
tr_enc = self.headers.get('transfer-encoding', '')
```